pkg/kubectl/resource_printer.go

```diff
@@ -880,13 +880,16 @@ func printIngress(ingress *extensions.Ingress, w io.Writer, options PrintOptions
 		}
 	}
 
-	if _, err := fmt.Fprintf(w, "%s\t%v\t%v\t%v\n",
+	if _, err := fmt.Fprintf(w, "%s\t%v\t%v\t%v",
```

Comment:
Originally this method is always called to append user specify columns at last, so a new line delimiter is necessary.